tests/integration/conftest.py

```diff
@@ -288,3 +289,68 @@ def factory(path: Path) -> ConfigUtils:
         return ConfigUtils(path)
 
     return factory
+
+
+@pytest.fixture
+def get_system_release(shell):
+    """
+    This function to be used whenever we need live information about system release.
+    E.g. after conversion system release check.
+    Otherwise, use hardcoded SYSTEM_RELEASE envar from /plans/main.fmf
+    This fixture returns a string of ID and VERSION_ID from /etc/os-release.
+    If /etc/os-release is not available, /etc/system-release is read instead.
+    These could be in generally used for OS specific conditioning.
+    Mapping of OS to ID:
+        {
+            "Centos Linux": "centos",\n
+            "Oracle Linux": "oracle",\n
+            "Alma Linux": "almalinux",\n
+            "Rocky Linux": "rocky"
+        }
+    Examples:
+        Centos Linux 7.9 => centos-7.9\n
+        Oracle Linux 8.6 => oracle-8.6\n
+        Alma Linux 8.4 => almalinux-8.4\n
+        Rocky Linux 8.5 => rocky-8.5
+    """
```

Comment:
I would write something like that for the docstrings, feel free to deny it if you think it's not worth it. 

```suggestion
    """
    This function to be used whenever we need live information about system release.
    E.g. after conversion system release check.
    Otherwise, use hardcoded SYSTEM_RELEASE envar from /plans/main.fmf
    This fixture returns a string of ID and VERSION_ID from /etc/os-release.
    If /etc/os-release is not available, /etc/system-release is read instead.
    These could be in generally used for OS specific conditioning.

    .. note::
        Mapping of OS to ID:
            {
                "Centos Linux": "centos",\n
                "Oracle Linux": "oracle",\n
                "Alma Linux": "almalinux",\n
                "Rocky Linux": "rocky"
            }

    .. examples::
        Centos Linux 7.9 => centos-7.9\n
        Oracle Linux 8.6 => oracle-8.6\n
        Alma Linux 8.4 => almalinux-8.4\n
        Rocky Linux 8.5 => rocky-8.5
    """
```

Reply:
I like yours more, thank you!